glossary.yml

```
@@ -4191,8 +4191,8 @@
   ja:
     term: パッケージ
     def: >
-      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
-      それに、何かの言語の場合は「[ライブラリー](#library)」と「[モジュール](#module)」と読んでいのこと。
+      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーション集合体。
```

Suggestion: 
配布や再利用が可能なコード、データおよびそれらに付随する文書をひとまとまりにしたもの。
or
 配布や再利用が可能なコード、データおよびそれらに付随する文書の集合体。

---

As mentioned above, I think ドキュメンテーション(documentation) is better for computer jargon. It seems to be used like this elsewhere in Japanese.

```suggestion
      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーションの集合体。
```

---

To clarify, in this case "[documentation](https://en.wikipedia.org/wiki/Software_documentation)" means manuals and other files to guide users or developers.